### DISCUSSION TITLE
Accessing codespace creators First and Last name via codespaces ENV variables

### DISCUSSION THREAD

**Discussion by TheLarkInn** [community/community#46783] [Codespaces] [answered]
### Select Topic Area

Question

### Body

As a part of being able to contribute to [rushstack](https://github.com/microsoft/rushstack), we require committers to set their local git config user name and email to look like the following: 

`> git config --local user.name "Sean Larkin"` 
`> git config --local user.email "thelarkinn@users.noreply.github.com"`

I am currently [revamping our codespaces dev container](https://github.com/microsoft/rushstack/pull/3966) for the project and I would like to automate this process for the user when they open a new codespace. I attempted to use `GIT_COMMITTER_EMAIL` and `GIT_COMMTTER_NAME` from [ENV](https://docs.github.com/en/codespaces/developing-in-codespaces/default-environment-variables-for-your-codespace) however they are both empty when I access them. I can use `GITHUB_USER` to automatically populate the `user.email`.

**Question:** Is there any way (via environment variables, or other settings) that I can fetch the codespace creator's first/last name so I can automatically setup these `git config` settings as a part of the postLoad scripts for the dev container.

**Comment by eljog** [comment, marked as answer]
Hi @TheLarkInn,
Codespaces always have the user.name and user.email set in the git config. They are set at `--system` level (not `--local`), and it should work for all git scenarios.
You can see them by running `git config --list` or `git config --list --system`.

**Reply by TheLarkInn** [reply]
Nice I'll experiment with this and see if I can assign the `git config user.name` to `git config --local user.name`!

**Reply by niksumeiko** [reply]
Did you managed to assign to —local?
